library/Cube/Web/IdoController.php

```diff
@@ -97,10 +103,6 @@ private function prepareCube(): void
             }
         }
 
-        foreach (['renderLayout', 'showFullscreen', 'showCompact', 'view'] as $p) {
-            $this->params->shift($p);
-        }
-
```

Comment:
Why this change?

Reply:
These are superfluous parameters, which now the CompatController already removes.

Reply:
:bulb: